app/static/grid_style.css

```diff
@@ -1,3 +1,22 @@
+:root[color-mode="regular"] {
+    --backgroundcol:#f7f7f7;
+    --textcolor:#222831;
+    --start-of-month:#e9ecef;
+    --primary-variant:#FFDE4D;
+    --secondary:#EF5454;
+    --borders:#E7E7E7;
+    --borders-variant:#F7F7F7;
+}
+
+:root[color-mode="dark"] {
+    --backgroundcol:#000000;
+    --textcolor:#EEEEEE;
+    --start-of-month:#8c28bf;
+    --secondary:#EF5454;
+    --borders:#E7E7E7;
+    --borders-variant:#F7F7F7;
+}
+
```

Comment:
Use prettifier to follow CSS best practices (capitalizing color's hex, spaces after `:` etc.)